Source/JavaScriptCore/dfg/DFGGraph.h

```diff
@@ -778,59 +788,43 @@ class Graph final : public virtual Scannable {
     {
         return NaturalBlockIterable(*this);
     }
-    
+
+    template<typename Functor>
+    class ForwardingFunc {
+    public:
+        explicit ForwardingFunc(const Functor& functor)
+            : m_functor(functor)
+        {
+        }
+
+        template<typename NodeType>
+        ALWAYS_INLINE auto operator()(NodeType*, Edge& edge) const -> decltype(std::declval<Functor>()(edge))
+        {
+            return m_functor(edge);
+        }
+
+    private:
+        const Functor& m_functor;
+    };
+
     template<typename ChildFunctor>
-    ALWAYS_INLINE void doToChildrenWithNode(Node* node, const ChildFunctor& functor)
+    ALWAYS_INLINE void doToChildren(Node* node, const ChildFunctor& functor)
```

Comment:
I think this can be `ALWAYS_INLINE void doToChildren(Node* node, const Invocable<IterationStatus(Node*)> auto& functor)`

Reply:
It seems armv7 complains about the missing template arg. So, just use the current approach.